### ISSUE TITLE
Pressable: onPress is not called for ctrl+click on Windows

### ISSUE THREAD

**Issue by tolfino** [necolas/react-native-web#2000]
**Is your feature request related to a problem? Please describe.**
Currently, Pressable takes over onClick and rejects all middle/Ctrl+click events. There is currently no way to handle these types of clicks if a Pressable is being used.

**Describe a solution you'd like**
I'm not really sure: perhaps allowing an onClick prop to be passed through Pressable to the underlying view would be the path of least resistance.

**Describe alternatives you've considered**
Using `View` directly supports `onClick` but does not work on mobile.

**Additional context**
On some websites, e.g. the Twitter site, middle or Ctrl+clicking a Pressable-looking element opens up a new tab. I'd like to be able to replicate this behavior.

**Comment by tolfino** [comment]
Unless I'm misunderstanding, https://github.com/necolas/react-native-web/blob/d8f60b4824e66053dc646b132e9ace7c607663f5/packages/react-native-web/src/modules/usePressEvents/PressResponder.js#L395 seems to explicitly reject clicks with `ctrlKey` and stops `onPress` from firing if its held, and there appears to be no way to get `onPress` to be called if `ctrlKey` is true.

**Comment by necolas** [comment]
That's if ctrlKey AND altKey are pressed, because that opens the native context menu.

**Comment by tolfino** [comment]
The logic is encoded incorrectly if that is the intention. Example code: https://codesandbox.io/s/divine-mountain-t04h4

Assuming `onPress != null`, if `ctrlKey = true` and `altKey = false`:

```javascript
onPress != null && event.ctrlKey === false && event.altKey === false  // implies
true && true === false && false === false  // implies
true && false && true  // implies
false
```

which causes the branch to be not taken.

If the intent is to not call `onPress` if both `ctrlKey` and `altKey` are held, the logic should be `onPress != null && (event.ctrlKey === false || event.altKey === false)`.

**Comment by necolas** [comment]
ctrl + click opens the native context menu on mac. alt + click downloads the resource. so i'm assuming you're on windows and want ctrl + click on a pressable link to trigger onPress? if you'd opened a bug report that would have asked you to include the OS and we wouldn't have gone around in circles trying to understand what the issue is

**Comment by tolfino** [comment]
Yes: sorry for opening the wrong type of issue. I assumed this behavior for Pressable was intentional and wanted to ask about adding a feature to handle these events that remained compatible with existing expected behavior.

**Comment by necolas** [comment]
Can you confirm you're experiencing this on windows?

As an aside, you can always use the component refs to attach native event listeners if you need to do something custom.

**Comment by tolfino** [comment]
Yes, on Windows the Ctrl+click event is swallowed and onPress does not fire.

Thanks for the tip about using component refs, that might also work.

**Comment by necolas** [comment]
This should be fixed in 0.16.2
